### PR TITLE
Update net.i2p.crypto.eddsa to 0.2.0

### PR DESCRIPTION
**Pull request by hagau**
Update net.i2p.crypto.eddsa to 0.2.0 to allow loading keys with the new OIDs from draft-ietf-curdle-pkix.
This overrides dependency of net.vrallev.ecc.ecc-25519-java on eddsa-0.1.0.

[net.vrallev.ecc.ecc-25519-java](https://github.com/vRallev/ECC-25519) is probably not going to get updated anytime soon, since it has been deprecated.

### CODE REVIEW

**Comment by jellelicht** [comment]
Could this and #34 be merged? The CI-failures seem related to old Java version and not indicative of any real issues.

**Comment by jplitza** [comment]
I have two questions:

1. Is this at all related to the interaction with OpenKeychain? It doesn't look that way to me.
2. Is this really cleaner? Now we have an constant defined at three different places, where previously we had the same constant in all three places.

**Comment by hagau** [comment]
@jplitza 
1. Yes, this is necessary for proper EdDSA support since OpenKeychain uses net.i2p.crypto.eddsa-0.2.0, which doesn't allow generation of EdDSA keys with the old OID, only import. (I think that was the reason, it's been a while) 
2. Thanks, I somehow overlooked that. I've moved the constant into Ed25519Verify.

File: sshlib/src/main/java/com/trilead/ssh2/signature/Ed25519Verify.java [hunk 1]
```diff
@@ -74,8 +76,7 @@ public static EdDSAPublicKey decodeSSHEd25519PublicKey(byte[] key) throws IOExce
 			throw new IOException("Ed25519 was not of correct length: " + keyBytes.length + " vs " + ED25519_PK_SIZE_BYTES);
 		}
 
-		return new EdDSAPublicKey(new EdDSAPublicKeySpec(keyBytes,
-				EdDSANamedCurveTable.getByName(EdDSANamedCurveTable.CURVE_ED25519_SHA512)));
+		return new EdDSAPublicKey(new EdDSAPublicKeySpec(keyBytes, EdDSANamedCurveTable.getByName(ED25519_CURVE_NAME) ));
```
Comment (kruton) [inline sshlib/src/main/java/com/trilead/ssh2/signature/Ed25519Verify.java]: nit: extra space between closing parathenses.

**Comment by dschuermann** [comment]
@kruton I was wondering about the CLA. Is it new?

Can you put a LICENSE file in the sshlib repo to clarify the status of this library? Assuming from some headers, it should be Apache v2.

**Comment by kruton** [comment]
Ah, I should fix any headers... probably IDE auto-inserted. It's 3-Clause BSD originally.

**Pull request merged**